frontend/app/layouts/static-page.vue

```diff
@@ -1,3 +1,13 @@
+<script setup lang="ts">
+import { setFavicon } from '~/utils/setFavicon'
+
+useOgImage()
+
+onMounted(() => {
```

Comment:
I think it would be better to move it to the `app.vue` and copy it to `error.vue` so we'll have only 2 duplicates, not amount of layouts